target/linux/tegra/base-files/lib/tegra.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
Drop this file. /tmp/sysinfo/model should be filed with the model name from the device tree source file by default.